### PR TITLE
Extend Spec Inheritance for remaining components

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- JITServer
- Build Node Lables
- OpenSSL
- Build Environment
- RELEASE variable
- Configure options
- Freemarker
- Repository Cache

[skip ci]
Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
I think building the JITServer specs should test most of the changes.
Jenkins compile xlinuxjit,plinuxjit,zlinuxjit, jdk8

**Comment by AdamBrousseau** [comment]
Jenkins compile osx jdk8,jdk11

**Comment by AdamBrousseau** [comment]
@vsebe for review

**Comment by AdamBrousseau** [comment]
Jenkins compile xlinux jdk8

**Comment by AdamBrousseau** [comment]
Jenkins compile xlinux jdk8,jdk13

**Comment by AdamBrousseau** [comment]
Jenkins compile osx jdk8,jdk11

**Comment by AdamBrousseau** [comment]
Jenkins compile osx jdk8,jdk11

**Comment by AdamBrousseau** [comment]
@vsebe I believe this is ready again for review.
Note this is blocking [JDK14](#7605)

**Comment by pshipton** [comment]
jenkins compile win,win32,xlinux,aix,plinux jdk8,jdk11

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -114,49 +114,39 @@ cmake:
...
     13: '/usr/lib/jvm/adoptojdk-java-12'
     next: '/usr/lib/jvm/adoptojdk-java-13'
   release:
+    all: 'linux-ppc64le-server-release'
     8: 'linux-ppc64le-normal-server-release'
     11: 'linux-ppc64le-normal-server-release'
-    12: 'linux-ppc64le-server-release'
-    13: 'linux-ppc64le-server-release'
-    next: 'linux-ppc64le-server-release'
   freemarker: '/home/jenkins/freemarker.jar'
```
Comment (vsebe) [inline buildenv/jenkins/variables/defaults.yml]: `freemarker` and `openjdk_reference_repo` could also be inherited. They have the same values for all specs except win and osx

**Comment by AdamBrousseau** [comment]
jenkins compile plinux,osx jdk11

**Comment by pshipton** [comment]
My previous PR build passed https://ci.eclipse.org/openj9/job/PullRequest-OpenJ9/2384/

**Comment by pshipton** [comment]
This should be squashed before merge.

**Comment by AdamBrousseau** [comment]
Sqashed. My last lest for freemarker and repo cache also passed.

**Pull request merged**